### PR TITLE
QA fixes round 2

### PR DESCRIPTION
**Pull request by DhaaraniCIT**

### CODE REVIEW

File: src/app/shared/components/apps/email-multi-select-field/email-multi-select-field.component.ts [hunk 1]
```diff
@@ -80,6 +86,9 @@ export class EmailMultiSelectFieldComponent implements OnInit {
     }
     this.addEmailForm.reset();
     this.showDialog = false;
+    if (this.appName === AppName.GUSTO) {
```
Comment (ashwin1111) [inline src/app/shared/components/apps/email-multi-select-field/email-multi-select-field.component.ts:89]: let's do this for all the apps, doesn't make sense to do it only for gusto

**Review by ashwin1111 — CHANGES_REQUESTED** [review]

File: src/app/shared/components/apps/email-multi-select-field/email-multi-select-field.component.html [hunk 2]
```diff
@@ -66,29 +63,39 @@ <h5 class="!tw-text-faded-text-color tw-text-14-px tw-pt-8-px !tw-font-400 !tw-l
...
+    <div [formGroup]="addEmailForm">
+        <div class="tw-p-24-px tw-border-b-1-px tw-border-b-box-color">
+            <div class="tw-flex tw-justify-between tw-items-center">
+                <p class="tw-text-slightly-normal-text-color tw-text-24-px tw-font-500">Add new Email Address</p>
+                <i class="pi pi-times tw-cursor-pointer" style="font-size: 12px" (click)="closeModel()"></i>
+            </div>
+        </div>
+
+        <div class="tw-p-24-px">
+            <p *ngIf="appName !== AppName.GUSTO" class="tw-pb-10-px tw-text-faded-text-color tw-text-12-px">
```
Comment (ashwin1111) [inline src/app/shared/components/apps/email-multi-select-field/email-multi-select-field.component.html:75]: why are we hiding this for gusto?

Comment (DhaaraniCIT) [reply src/app/shared/components/apps/email-multi-select-field/email-multi-select-field.component.html:75]: The design team wants to hide it

Comment (ashwin1111) [reply src/app/shared/components/apps/email-multi-select-field/email-multi-select-field.component.html:75]: can you point me to the message link?

Comment (DhaaraniCIT) [reply src/app/shared/components/apps/email-multi-select-field/email-multi-select-field.component.html:75]: they reported it that it should be as in design, thats why

Comment (DhaaraniCIT) [reply src/app/shared/components/apps/email-multi-select-field/email-multi-select-field.component.html:75]: https://fylein.slack.com/archives/C04GGNXSEAW/p1680602711419119?thread_ts=1680514080.584629&cid=C04GGNXSEAW

File: src/app/shared/components/apps/email-multi-select-field/email-multi-select-field.component.ts [hunk 3]
```diff
@@ -2,9 +2,10 @@ import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
 import { FormBuilder, FormGroup, Validators } from '@angular/forms';
 import { DropdownFilterOptions } from 'primeng/dropdown';
 import { BambooHRConfigurationPost } from 'src/app/core/models/bamboo-hr/bamboo-hr.model';
-import { ClickEvent } from 'src/app/core/models/enum/enum.model';
+import { AppName, ClickEvent, ToastSeverity } from 'src/app/core/models/enum/enum.model';
```
Comment (ashwin1111) [inline src/app/shared/components/apps/email-multi-select-field/email-multi-select-field.component.ts:5]: AppName is unused

**Review by ashwin1111 — APPROVED** [review]

**Pull request merged**